packages/core/src/browser/preferences/preference-contribution.ts

```diff
@@ -131,7 +135,7 @@ export class PreferenceSchemaProvider extends PreferenceProvider {
     }
 
     protected doSetSchema(schema: PreferenceSchema): PreferenceProviderDataChange[] {
-        const scope = this.getScope();
+        const scope = PreferenceScope.Default;
```

Comment:
is this something you are going to change in near future? With this change the "scope"s of all prefs will be the same

Reply:
How can it be anything except default in default preference schema provider?

Reply:
you are right. i misunderstood